build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         // In general, we should aim to keep these in sync with AC
         compileSdkVersion: 34,
         targetSdkVersion: 34,
-        minSdkVersion: 21,
+        minSdkVersion: 24,
```

Comment:
This can be downgraded again. It failed locally somehow, so I hope just a local settings error.